src/components/SpecialResolution/Resolution.vue

```diff
@@ -185,7 +188,12 @@ export default class Resolution extends Vue {
 
   /** For cancel/undo button, stores using setSpecialResolution. */
   async undoSpecialResolutionStore (): Promise<void> {
-    this.isEditing = false
+    // Remain in edit mode, if special resolution filing.
+    if (!this.isCoopCorrectionFiling) {
```

Comment:
Would it make sense to create a `isCoopSpecialResolution` getter? It's usually better to check for "in" conditions than "out" conditions (esp in case we add entity types or filing types).

Reply:
`isSpecialResolutionFiling`